source/faq.rst

```diff
@@ -1,264 +1,219 @@
 
-Any modern 64-bit Linux distribution. It is tested on the latest
-versions of Debian, Ubuntu, CentOS, and Red Hat Enterprise Linux.
 
-Minimum 100 MB of storage is required for installing the |pmm-client|
-package.  With good constant connection to |pmm-server|, additional
-storage is not required.  However, the client needs to store any
-collected data that it is not able to send over immediately, so
-additional storage may be required if connection is unstable or
-throughput is too low.
 
 .. _data-retention:
+.. _how-to-control-data-retention-for-pmm:
 
+**************************************
 How to control data retention for PMM?
-================================================================================
+**************************************
 
-By default, both |prometheus| and QAN store time-series data for 30 days.
+By default, both Prometheus and QAN store time-series data for 30 days.
 
-Depending on available disk space and your requirements,
-you may need to adjust data retention time.
+Depending on your available disk space and requirements, you may need to adjust the data retention time.
 
 You can control data retention by the following way.
 
-#. Select the |pmm-settings| dashboard in the main menu.
+#. Select the PMM Settings dashboard in the main menu.
+
+   .. image:: /.res/graphics/png/pmm-add-instance.png
+
+#. In the *Settings* section, enter the new data retention value in days.
 
-   .. figure:: .res/graphics/png/pmm-add-instance.png
+   .. image:: /.res/graphics/png/pmm.settings_settings.png
 
-      Choosing the |pmm| *Settings* menu entry
+#. Click *Apply changes*.
 
-#. In the *Settings* section, enter new data retention value in days.
 
-   .. figure:: .res/graphics/png/pmm.settings_settings.png
 
-      Entering data retention on the *Settings dashboard*
+***********************************************
+How often are NGINX logs in PMM Server rotated?
+***********************************************
 
-#. Click the *Apply changes* button.
+PMM Server runs ``logrotate`` on a daily basis to rotate NGINX logs and keeps up to ten of the most recent log files.
 
-How often are nginx logs in PMM Server rotated?
-================================================================================
 
-|pmm-server| runs ``logrotate`` to rotate nginx logs on a daily basis
-and keep up to 10 latest log files.
 
 .. _privileges:
 
-What privileges are required to monitor a |mysql| instance?
-================================================================================
+*********************************************************
+What privileges are required to monitor a MySQL instance?
+*********************************************************
+
+.. code-block::
+
+   GRANT SELECT, PROCESS, SUPER, REPLICATION CLIENT, RELOAD ON *.* TO 'pmm'@'localhost';
+   GRANT SELECT ON performance_schema.* TO 'pmm'@'localhost';
```

Comment:
This line is redundant as we already grant SELECT ... *.* in the earlier privilege. I suggest you remove it.

Reply:
Thanks. I'd copied it from https://www.percona.com/doc/percona-monitoring-and-management/2.x/concepts/services-mysql.html#pmm-conf-mysql-user-account-creating
where I will change it too.